### PR TITLE
hdiutil: add page

### PR DESCRIPTION
**Pull request by dpinela**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

**Comment by waldyrious** [comment]
Nice contribution, thanks! I have nothing to point out in the contents, but maybe consider adding a URL, if there's one we could use. Otherwise, LGTM! 👍

**Comment by dpinela** [comment]
> Nice contribution, thanks! I have nothing to point out in the contents, but maybe consider adding a URL, if there's one we could use. Otherwise, LGTM! 👍

This is a built-in utility in macOS. I could add a link to the man page (though I can't find them on Apple's website) but none of the others seemed to have it, so I did the same.

**Comment by waldyrious** [comment]
Yeah, many pages were created before we introduced the URL field. But sure, I suspected that a link wouldn't be available. Apple used to host manpages in the past but for some reason they discontinued that practice :/

Given that, I think this is ready to go. Thanks!

File: pages/osx/hdiutil.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# hdiutil
+
+> Utility to create and manage disk images.
+
+- Mount an image:
+
+`hdiutil attach {{filename}}`
```
Comment (agnivade) [inline pages/osx/hdiutil.md]: All of the tokens should be snake_case, like file_name, volume_name.

File: pages/osx/hdiutil.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+`hdiutil detach /Volumes/{{volumename}}`
+
+- List mounted images:
+
+`hdiutil info`
+
+- Create an ISO image from the contents of a directory:
+
+`hdiutil makehybrid -o {{filename}} {{dirname}}`
```
Comment (agnivade) [inline pages/osx/hdiutil.md]: dirname should be path/to/directory

File: pages/osx/hdiutil.md (hunk 1 above)
Comment (sbrl) [reply pages/osx/hdiutil.md]: I think `filename` is ok as-is - I believe we've done `path/to/filename` before though.

Comment (agnivade) [reply pages/osx/hdiutil.md]: Yeah ok, I see a lot of `filename`s. But we should change dirname to path/to/directory though. There are no other `dirname`s

**Pull request merged**